docker/web/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM node:16-buster AS builder
+
+# Install doppler CLI
+RUN (curl -Ls --tlsv1.2 --proto "=https" --retry 3 https://cli.doppler.com/install.sh || wget -t 3 -qO- https://cli.doppler.com/install.sh) | sh -s -- --verify-signature
+
+# node images create a non-root user that we can use
+USER node
+WORKDIR /home/node/freeCodeCamp
+
+# Pass `DOPPLER_TOKEN` at build time to create an encrypted snapshot for high-availability
```

Comment:
Is this just in case your internet is flaky?

Reply:
Yes - because we will reference the keys during the build, it's best to fail early if Doppler is unreachable. 

That way we don't waste time installing dependencies.